frontend/src/uxbox/main/ui/workspace/drawarea.cljs

```diff
@@ -74,6 +74,66 @@
     :name "Text"
     :content nil}])
 
+(defn- make-minimal-shape
+  [type]
+  (let [tool (seek #(= type (:type %)) minimal-shapes)]
+    (assert tool "unexpected drawing tool")
+    (assoc tool
+           :id (uuid/next)
+           :x 0
+           :x1 0
+           :x2 0
+           :y 0
+           :y1 0
+           :y2 0
```

Comment:
You don't need to add x1, x2, y1 and y2. These are fields only in selrect. Also, this parameters can break the application. It doesn't make sense a 0 width or height. The minimum should be 1

Reply:
fixed